src/shogun/machine/LinearMachine.cpp

```diff
@@ -120,3 +141,50 @@ void CLinearMachine::store_model_features()
 {
 }
 
+float64_t CLinearMachine::compute_bias(CFeatures* data)
+{
+	REQUIRE(m_labels,"No labels set\n");
+
+	if (!data)
+		data=features;
+
+	REQUIRE(data,"No features set\n");
+
+	REQUIRE(m_labels->get_num_labels() == data->get_num_vectors(),"Number of training vectors does not match number of labels\n");
+
+	SGVector<float64_t> outputs = apply_get_outputs(data);
+
+	int32_t num_vec=data->get_num_vectors();
+
+	Map<VectorXd> eigen_outputs(outputs,num_vec);
+	Map<VectorXd> eigen_labels(((CRegressionLabels*)m_labels)->get_labels(),num_vec);
+
+	return (eigen_labels - eigen_outputs).mean() ;
```

Comment:
I checked linalg. I don't think there is mean computation.There is sum,max,dot,...etc but there is no mean.